src/components/Messages/MessagePreview.jsx

```diff
@@ -60,7 +69,14 @@ const MessagePreview = ({ message, folderType }) => {
           </Box>
         </Box>
       )}
-      {replyMessage && <NewMessage oldMessage={message} closeForm={() => setReplyMessage(false)} />}
+      {replyMessage && (
+        <NewMessageModal
+          showModal={showModal}
+          setShowModal={setShowModal}
+          oldMessage={message}
+          closeForm={() => setReplyMessage(false)}
```

Comment:
I can't find anything about `closeForm`, I don't think it's a valid prop - but could be wrong.

I believe what is intended from this line is to run this function when the `NewMessageModal` closes.  IF that is true, this can go in the `onClose` function within `NewMessageModal.jsx` - line 110.